modules/raft/src/main/java/org/apache/ignite/raft/jraft/StateMachine.java

```diff
@@ -102,8 +102,10 @@ public interface StateMachine {
      * {@link #onConfigurationCommitted(Configuration)} as full configuration entry is provided.
      *
      * @param conf committed configuration
+     * @param lastAppliedIndex Last applied index.
+     * @param lastAppliedTerm Last applied term.
      */
-    default void onRawConfigurationCommitted(ConfigurationEntry conf) {
+    default void onConfigurationCommittedWithLastAppliedIndexAndTerm(ConfigurationEntry conf, long lastAppliedIndex, long lastAppliedTerm) {
```

Comment:
Same thing about renaming: it seems that the old name is ok as the parameters already explain what they are for

Reply:
fixed